src/components/Books/BooksActions.js

```diff
@@ -6,9 +6,11 @@ export const LoadBooks = () => {
   return (dispatch, getState) => {
     let pageNumber = getState().PaginationReducer.page;
     let pageSize = getState().PaginationReducer.pageSize;
+    let title = getState().PaginationReducer.title;
+    let price = getState().PaginationReducer.price;
     return axios
       .get(
```

Comment:
Axios get params should be in separate param. https://github.com/axios/axios#example